js/app/application.js

```diff
@@ -88,6 +89,8 @@ const Application = new Knowledge.Class({
                              'Path to a overrides scss or css file to theme the application', null);
         this.add_main_option('app-json-path', 'J'.charCodeAt(), GLib.OptionFlags.NONE, GLib.OptionArg.FILENAME,
                              'Path to a yaml or json file to use as a preset', null);
+        this.add_main_option('moltres-content', 'M'.charCodeAt(), GLib.OptionFlags.NONE, GLib.OptionArg.NONE,
```

Comment:
Not to argue against our word ladder name, but `dummy-content` would probably be a clearer option name here. Especially as we are opening up ekl and documenting all out things.